deny.toml

```diff
@@ -1,22 +1,33 @@
 [licenses]
+unlicensed = "deny"
+default = "deny"
+copyleft = "deny"
 allow = [
-  "MIT",
+  "0BSD",
+  "Apache-2.0",
+  "BSD-2-Clause",
+  "BSD-3-Clause",
+  "BSL-1.0",
   "CC0-1.0",
   "ISC",
+  "MIT",
   "OpenSSL",
-  "Unlicense",
```

Comment:
Could you expand why this was removed? The "Unlicense" is effectively a public-domain declaration, which is different than unlicensed.

Reply:
Ah, yeah, I removed it because we don't have any dependencies using it. It would be harmless to leave in though.